packages/data/src/utils/emitter.ts

```diff
@@ -1,12 +1,20 @@
+export interface DataEmitter {
+	emit: VoidFunction;
+	subscribe: ( listener: VoidFunction ) => VoidFunction;
+	pause: VoidFunction;
+	resume: VoidFunction;
+	isPaused: boolean;
+}
+
 /**
  * Create an event emitter.
  *
- * @return {import("../types").DataEmitter} Emitter.
+ * @return {DataEmitter} Emitter.
```

Comment:
In TypeScript files I recommend to not include the type in the JSDoc annotation because it's duplicated. It should be enough to just document the value.

Reply:
Done. Thanks